src/__test__/redux/reducers/componentConfig/loadPlotData.test.js

```diff
@@ -0,0 +1,76 @@
+import fetchMock, { enableFetchMocks } from 'jest-fetch-mock';
+import { act } from 'react-dom/test-utils';
+
+import { makeStore } from 'redux/store';
+
+import { plotTypes } from 'utils/constants';
+import fake from '__test__/test-utils/constants';
+
+import { fetchPlotDataWork, loadPlotConfig } from 'redux/actions/componentConfig';
+import pushNotificationMessage from 'utils/pushNotificationMessage';
+import { fetchWork } from 'utils/work/fetchWork';
+
+enableFetchMocks();
+
+// Return 404 to use default plot config
+fetchMock.mockResponse(() => ({
+  status: 404,
+  body: 'Not Found',
+}));
+
+jest.mock('utils/pushNotificationMessage');
+jest.mock('utils/work/generatePlotWorkBody');
+jest.mock('utils/work/fetchWork', () => ({
+  fetchWork: jest.fn(() => Promise.resolve(
+    ['work-result-1', 'work-result-2'],
+  )),
+}));
+
+const experimentId = fake.EXPERIMENT_ID;
+let testStore = null;
+
+const plotUuid = 'DotPlotMain';
+const plotType = plotTypes.DOT_PLOT;
+
+describe('plotDataLoaded', () => {
```

Comment:
`loadPlotData` is changed to `fetchPlotDataWork`. Change the test name accordingly

Reply:
changed